lib/req-res/status.js

```diff
@@ -32,16 +25,24 @@ module.exports = function ( options ) {
 				'code'    : code || 0 // optional
 			} );
 
+			log.error( data, {
+				'data'        : errorData,
+				'code'        : code,
+				'messageType' : options.messageType,
+				'msg'         : options.message
+			} );
+
 		},
 
 		'fail' : function ( data ) {
 
-			log.warn( data );
-
 			respond( {
 				'status' : 'fail',
 				'data'   : data
 			} );
+
+			log.warn( data );
```

Comment:
Should we add the `options.messageType` and possibly `options.message` here too? It would be nice to see what messageType the fail came from.